dev/ci/ci-iris.sh

```diff
@@ -9,13 +9,15 @@ git_download iris_string_ident
 git_download iris_examples
 
 # Extract required version of Iris (avoiding "+" which does not work on MacOS :( *)
-iris_CI_REF=$(grep -F '"coq-iris"' < "${CI_BUILD_DIR}/iris_examples/coq-iris-examples.opam" | sed 's/.*"dev\.[0-9][0-9.-]*\.\([0-9a-z][0-9a-z]*\)".*/\1/')
+iris_CI_REF=$(grep -F '"coq-iris-heap-lang"' < "${CI_BUILD_DIR}/iris_examples/coq-iris-examples.opam" | sed 's/.*"dev\.[0-9][0-9.-]*\.\([0-9a-z][0-9a-z]*\)".*/\1/')
+[ -n "$iris_CI_REF" ] || { echo "Could not find Iris dependency version" && exit 1; }
```

Comment:
Something doesn't seem to work about this combinator... and anyway wouldn't it be more readable to do
```
if [ -z "$iris_CI_REF" ]; then
  echo "Could not find Iris dependency version"
  exit 1
fi
```

Reply:
It works though https://gitlab.com/coq/coq/-/jobs/845135052

Reply:
I think it's the previous version of the PR which had some issue. This one seems to work fine.

Nonetheless, I agree with the readability concern.